assets/views/home/header.ejs

```diff
@@ -33,6 +33,7 @@
             <li><a href="#" data-toggle="modal" data-target="#flairAppList">Flair App List</a></li>
             <li><a href="#" data-toggle="modal" data-target="#flairMod">Edit Flairs</a></li>
             <li><a href="/mod/banuser">Ban a User</a></li>
+            <li><a target="_self" href="/mod/downloadModmail" download="modmails.txt">Download Modmails</a></li>
```

Comment:
Why have we got this here?

I had thought about adding it as an option when the mod is searching. so show all the results, and also do a quick search through this .txt file and allow them to download it if it has some results in it.

Means less steps for the mod.